include/dsVideoPort.h

```diff
@@ -188,7 +188,8 @@ dsError_t  dsIsVideoPortEnabled(intptr_t handle, bool *enabled);
 /**
  * @brief Checks whether the specific video port is connected to display.
  * 
- * This function is used to check whether video port is connected to a display or not.
+ * For source devices, this function is used to check whether video port is connected to a display or not.
+ * For sink devices, this function returns the display connection status as true always. Please refer :: dsVIDEOPORT_TYPE_INTERNAL
```

Comment:
Rephrase the sentence to "For Sink devices, where the display is always connected (dsVIDEOPORT_TYPE_INTERNAL), this API will consistently return true for the display connection status."

Reply:
Updated